Kudu.Core/Scripts/kudusync

```diff
@@ -1,4 +1,4 @@
 #!/bin/sh
-"$0.NET.exe" "$@"
+mono "$0.NET.exe" "$@"
```

Comment:
I see. What about checking OS type, then? If it's Unix then prepend mono at the line. Does this logic sound right?